ssl/statem/statem_clnt.c

```diff
@@ -2454,7 +2450,6 @@ MSG_PROCESS_RETURN tls_process_new_session_ticket(SSL *s, PACKET *pkt)
     if (ticklen == 0)
         return MSG_PROCESS_CONTINUE_READING;
 
-    /* TODO(TLS1.3): Is this a suitable test for TLS1.3? */
```

Comment:
I guess I didn't read far enough further down into the function then (re session_id use to detect resumption).

I am trying to replay your argument on the 1-1 correspondence question for a rephrased question about a given SSL_SESSION having at most one ticket correspond to it, and it is not really applying.  Am I shooting at the wind here?